dev-ruby/with_advisory_lock/with_advisory_lock-5.1.0.ebuild

```diff
@@ -27,6 +27,8 @@ ruby_add_bdepend "test? (
 	>=dev-ruby/activerecord-6.1:*[sqlite]
 	dev-ruby/maxitest
 	dev-ruby/mocha
+	dev-ruby/sqlite3:0
+	dev-ruby/yard
```

Comment:
I would assume that this dependency is then also needed with USE=doc? I see that RUBY_FAKEGEM_RECIPE_DOC is set to "none" but the yard dependency seems to indicate that documentation can be build and installed.